kong/plugins/datadog/handler.lua

```diff
@@ -28,10 +28,11 @@ local get_consumer_id = {
 }
 
 
-local function compose_tags(service_name, status, consumer_id, tags, conf)
+local function compose_tags(service_name, status, route_name, consumer_id, tags, conf)
```

Comment:
I thought it looked weird coming after "conf" as the last argument, but yeah in case someone is importing this plugin and using the code, I have changed it.